package.json

```
@@ -10,8 +10,7 @@
     "build": "run-s build:*",
     "build:remix": "remix build",
     "build:server": "tsx ./other/build-server.ts",
-    "dev": "remix dev -c \"npm run dev:server\" --no-restart",
-    "dev:server": "cross-env MOCKS=true tsx watch --clear-screen=false --ignore \"app/**\" --ignore \"build/**\" --ignore \"node_modules/**\" --inspect ./index.js",
+    "dev": "remix dev -c \"cross-env MOCKS=true node --inspect ./server/dev-server.js\" --no-restart",
```

Let's move the environment variable to the execa can and the --inspect flag too. Then we can do:

```suggestion
    "dev": "remix dev -c \"node ./server/dev-server.js\" --no-restart",
```

---

done